src/ncp/spinel.h

```diff
@@ -2003,6 +2003,23 @@ typedef enum {
      */
     SPINEL_PROP_CHILD_SUPERVISION_CHECK_TIMEOUT = SPINEL_PROP_OPENTHREAD__BEGIN + 11,
 
+    /// Thread parent response info
```

Comment:
nit: Parent Response should be capitalized since it's an "official" Thread frame type.

Reply:
okay